### PR TITLE
Build index-side Components #2

### LINKED ISSUE #2: Build index-side Components
- [x] BurgerButton
- [x] Menu
- [x] PlayButton
- [x] InputButton

### LINKED ISSUE #3: Create index side
- [x] Get HTML structure
- [x] Set background image
- [x] Insert needed components
- [x] Styled all components

### PR DESCRIPTION
**Pull request by ShiningElektra**
#Issue 2
- Add Burger-Button
- Add Navigation-Component
- Get them clickable
- Styling Navigation-Component

https://recycool-app-shiningele-u7yowy.herokuapp.com/

### CODE REVIEW

**Review by mrana17 — APPROVED** [review]
lgtm 🚀

**Comment by SilviaBarcellini** [comment]
You did great! 🍔  🚀

File: components/burger/Burger.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import styles from "../burger/Burger.module.css";
+
+// export type BurgerProps = {
+//   primary: boolean;
+// };
+
+const Burger = () => {
+  return (
+    <div className={`${styles.burger}`}>
```
Comment (HorstFrank) [inline components/burger/Burger.tsx]:
```suggestion
    <div className={styles.burger}>
```
I saw that silvia did the same. doesn't it work without backticks?!?

**Review by HorstFrank — APPROVED** [review]
YEAH!

File: components/burger/Burger.tsx (hunk 1 above)
Comment (ShiningElektra) [reply components/burger/Burger.tsx]: danke :-)

File: components/menu/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styles from "../menu/Menu.module.css";
+import React from "react";
+
+export type MenuProps = { open: boolean };
+// property die open heißt
```
Comment (phlgr) [inline components/menu/Menu.tsx]: Remove comments

File: pages/index.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import Head from "next/head";
 import Button from "../components/buttons/Button";
 import styles from "../styles/Home.module.css";
+import React from "react";
```
Comment (phlgr) [inline pages/index.tsx]: You could remove this import

**Review by phlgr — COMMENTED** [review]
GJ 🚀

**Review by cboenke — APPROVED** [review]
GJ! 🚀

File: components/burger/Burger.module.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.burger {
+  position: absolute;
+  top: 4%;
+  right: 4%;
```
Comment (phlgr) [inline components/burger/Burger.module.css:4]: I would probably use rem here, as this translates to pixels and the percentage is actually dependent on the device width or height.

File: components/burger/Burger.module.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 1.8rem;
+  background: transparent;
+  cursor: pointer;
+  z-index: 10;
+  opacity: 0.5;
```
Comment (phlgr) [inline components/burger/Burger.module.css:12]: I also find i t quite hard to see the Burger once the Menu is open

File: components/menu/Menu.module.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
+.menu {
+  position: absolute;
+  top: 3%;
+  right: 2%;
```
Comment (phlgr) [inline components/menu/Menu.module.css:4]: Same as in Burger.module.css

**Review by phlgr — CHANGES_REQUESTED** [review]
GJ 🚀

File: components/burger/Burger.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  component: Burger,
+} as Meta;
+
+const Template: Story<BurgerProps> = (args) => <Burger {...args} />;
+
+export const Closed = Template.bind({});
+Closed.args = { open: false };
+
+export const Open = Template.bind({});
+Open.args = { open: true };
```
Comment (mewiemers) [inline components/burger/Burger.stories.tsx:17]: it is necessary to say true or false or can you set an boolean here?

**Review by mewiemers — COMMENTED** [review]
Lets recycle! LGTM! ♻

File: components/burger/Burger.stories.tsx (hunk 7 above)
Comment (ShiningElektra) [reply components/burger/Burger.stories.tsx:17]: If I understand it correctly this is the start attribute of the boolean inside of Burger.tsx

File: components/burger/Burger.module.css (hunk 4 above)
Comment (ShiningElektra) [reply components/burger/Burger.module.css:4]: Thx!

**Pull request merged**